deploy/cr.yaml

```diff
@@ -60,6 +60,8 @@ spec:
     size: 3
     image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0
     autoRecovery: true
+#    mySqlAllocator: jemalloc
```

Comment:
In cr we have a commented option with a default value. In our doc, we will add all possible values. Also, you can add them in CRD. It is possible to add validation on CRD level.